src/oceans/models/guide.js

```diff
@@ -49,12 +59,15 @@ const guides = [
       }
     },
     style: 'Info',
+    didYouKnow: true,
```

Comment:
i'm not sure we need this extra property on guides -- `guide.style === 'Info'` should work in place of `guide.didYouKnow`

(cc @breville to double-check this thought)

Reply:
I'm okay with that. I've made this change -- let me know if there are any objections.